src/app/models/product.ts

```diff
@@ -23,6 +23,9 @@ export class Product extends JsonApiModel {
   available: true;
 
   @Attribute()
+  dailydishfor: true;
```

Comment:
is `true` a data type?

Reply:
No, its so we can access the data being sent from the server.